Fridge_ingredients.sql

```diff
@@ -0,0 +1,37 @@
+CREATE ROLE tech_folks NOINHERIT LOGIN PASSWORD 'password';
+
+CREATE DATABASE Fridge_ingredients
+	WITH
+	OWNER = tech_folks
+	ENCODING = 'UTF8'
+	CONNECTION LIMIT = -1;
+
+CREATE TABLE Fridge_contents
```

Comment:
This almost perfect, just need to re-order the tables created so that the foreign keys can be used properly. The order should go create units table, create ingredients table and create fridge_contents table.

Reply:
I apologise for my mistake. Thank you for pointing out my mistake. I hope we could merge the queries into the repository.